pkg/node/utils.go

```diff
@@ -56,19 +57,37 @@ func checkDrive(drive *directcsi.DirectCSIDrive, volumeID string, probeMounts fu
 	return fmt.Errorf("drive %v is not mounted at mount point %v", drive.Name, mountPoint)
 }
 
-func checkStagingTargetPath(stagingPath string, probeMounts func() (map[string][]sys.MountInfo, error)) error {
+func checkMountPath(mountPath string, probeMounts func() (map[string][]sys.MountInfo, error)) error {
+
+	writeCheck := func(path string) error {
```

Comment:
This check doesn't make sense. You cannot ensure you are really writing to a mount point. This function will succeeded on unmounted target.

Reply:
This check will be done only after ensuring the mountpath is mounted correctly (entry found in mountinfo probe)